tasks/docker-compose.yml

```diff
@@ -15,6 +15,9 @@
 
 - name: Install Docker Compose (if configured).
   get_url:
-    url: https://github.com/docker/compose/releases/download/{{ docker_compose_version }}/docker-compose-Linux-x86_64
+    url: "{{ docker_compose_url }}"
     dest: "{{ docker_compose_path }}"
     mode: 0755
+  when: >
+    docker_compose_current_version.stdout is not defined
+    or docker_compose_version not in docker_compose_current_version.stdout
```

Comment:
Just to be clear, this is to make it so this task is _only_ run at all if the docker-compose command shows the version differs from the version desired?

Reply:
That's correct, @geerlingguy! I observed that prior to this commit, docker-compose was re-downloaded on subsequent runs despite already having the desired version (not idempotent). I have amended the commit for clarity, as the `docker-compose --version` task was also not previously constrained to use the docker-compose binary at `docker_compose_path`.